src/peerstate.rs

```diff
@@ -382,11 +382,11 @@ impl<'a> Peerstate<'a> {
             sql::execute(
                 self.context,
                 sql,
-		"UPDATE acpeerstates \
-		 SET last_seen=?, last_seen_autocrypt=?, prefer_encrypted=?, \
-		 public_key=?, gossip_timestamp=?, gossip_key=?, public_key_fingerprint=?, gossip_key_fingerprint=?, \
+                "UPDATE acpeerstates \
```

Comment:
fyi, i've been converting SQL strings to `concat!(...)` when encountering them which i think is even nicer.  I guess that's all subjective though

Reply:
I think `indoc!` would be nicest and least error prone (i once hunted a bug where a space was missing in an sql statement -- took me 2h or so).

Reply:
hmm, if that's the concern does it actually do anything that the existing string literal doesn't?  The only thing which actually annoys me about the existing ones is the escaping of the newline.  But is this actually needed?  So perhaps reformatting as here but also stripping the `\` is the way to go?

Reply:
iiuc indoc + escaping newlines could also lead to the missing space issue.

Reply:
i think, concat! can still easily produce problems, esp. when trying to format smartly:
```
concat!("  SELECT a",
        "    FROM b",
        "   WHERE c",
        "ORDER BY d")
```

would result in `  SELECT a    FROM b   WHERE cORDER BY d` - i think this is no win compared with the rust-literals. you have to take care that there is a space between the lines.

indoc! requires less boilerplate and keeps the original linebreaks - which are treated as normal whitespace in sqlite then:
```
indoc!("  SELECT a
            FROM b
           WHERE c
        ORDER BY d")
```

results in `  SELECT a\n    FROM b\n   WHERE c\nORDER BY d\n`

Reply:
ah, and it is hard to format things with the concat macro, i did some tests at https://github.com/deltachat/deltachat-core-rust/pull/852